src/interaction/sessionrecording.cpp

```diff
@@ -409,16 +410,22 @@ bool SessionRecording::startPlayback(std::string& filename,
         LERROR("Unknown data type in header (should be Ascii or Binary)");
         cleanUpPlayback();
     }
-    // throwaway newline character
-    readHeaderElement(_playbackFile, 1);
+    // throwaway newline character(s)
+    std::string lineEnd = readHeaderElement(_playbackFile, 1);
+    bool isDosLineEnding = (lineEnd == "\r");
+    if (isDosLineEnding) {
+        // throwaway the second newline character (\n) also
+        readHeaderElement(_playbackFile, 1);
+    }
 
     if (_recordingDataMode == DataMode::Binary) {
```

Comment:
Are you sure we are actually writing the \r in the case of a binary session recording file?  I thought we'd write strings in this case as a length + the exact number of characters

Reply:
It's true that when we write the header it has only `\n`. But this could get changed by doing a unix2dos conversion, or modifying an ascii format recording in a text editor.
I did remove the check for the `\r` line ending in the previous comment, though, because I don't see any reasonable way that problem could happen with a binary format recording.